app/views/comments/cud.js.erb

```diff
@@ -0,0 +1,6 @@
+$("#comments-<%= @reference.id %>").html("<%= escape_javascript(render 'papers/comments', reference: @reference) %>");
```

Comment:
Instead of a class variable, let's have the partial take it as a local variable. Makes it easier to reuse later - you don't need to set up the controller to have a `@reference` variable, you just feed a reference into the partial.